<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTextWidget</class>
 <widget class="QWidget" name="SearchTextWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>24</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>262</width>
    <height>24</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>376</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="exactlyCheckBox">
     <property name="minimumSize">
      <size>
       <width>58</width>
       <height>17</height>
      </size>
     </property>
     <property name="text">
      <string>exactly</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/Cubator.qrc">
       <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string>Result</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/Cubator.qrc">
       <normaloff>:/icons/left.png</normaloff>:/icons/left.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../resources/Cubator.qrc">
       <normaloff>:/icons/right.png</normaloff>:/icons/right.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources/Cubator.qrc"/>
 </resources>
 <connections/>
</ui>
